irohad/consensus/yac/impl/yac_gate_impl.hpp

```diff
@@ -49,6 +49,10 @@ namespace iroha {
                     std::shared_ptr<network::BlockLoader> block_loader,
                     uint64_t delay);
         void vote(const shared_model::interface::Block &) override;
+        /**
```

Comment:
Please format this as it is done https://github.com/hyperledger/iroha/pull/1252/files#diff-602f45921f2f767d49042cbc8221d796R27 here, with `@return`.